coderd/database/migrations/000274_workspace_modules.up.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE
+    workspace_resources
+ADD
+    COLUMN module_path TEXT;
+
+CREATE TABLE workspace_modules (
+    id uuid NOT NULL,
+    job_id uuid NOT NULL REFERENCES provisioner_jobs (id) ON DELETE CASCADE,
+    transition workspace_transition NOT NULL,
```

Comment:
Do we need to capture the module for all transitions? What's the value of this?

Reply:
I [mirrored](https://github.com/coder/coder/blob/ea2b6b7cbc9022ec92ee49f74c75aefd77613360/coderd/database/dump.sql#L1688) `workspace_resources`. I think for template import jobs you get resources that reference the same provisioner job but have different workspace transitions. My thinking was if you'd like to associate a specific resource from a template import job with its module, you'd need to know which transition it came from.